src/index.ts

```diff
@@ -134,6 +139,14 @@ async function main() {
   await program.parseAsync();
 }
 
+function parseIntOption(option: string, _value: string) {
+  const parsed = Number(option);
```

Comment:
why not delegint the `_value` if we are not using? does it give you an error?

Reply:
Because I'm silly and always think of absolute function signatures. A curse of solidity :see_no_evil: (removed `_value` and it's not complaining).